### PR TITLE
Update LICENSE and Dependabot

### PR DESCRIPTION
**Pull request by ncalteen**
This PR does the following:

- Updates `LICENSE` to GitHub's standard MIT license
- Changes the Dependabot configuration to run weekly
- Removes the additional labels added by Dependabot (as they may not exist in developers' new repos)

### CODE REVIEW

**Review by sethrylan — APPROVED** [review]
LGTM

**Comment by promiseofcake** [comment]
Question, given that sometimes dependabot will update a dependency that updates the dist/ file, how should we go about fixing those PRs? Pull down and re-commit? Is there any reason why we wouldn't just have the action itself generate / merge the dist directory on each run?

**Comment by ncalteen** [comment]
Hey @promiseofcake, I wanted to take some time to dig into this and test out a few approaches. Basically end result is it is certainly possible, but there are some different issues that would arise and the solutions aren't that great at the moment.

From the [GitHub Actions docs](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#using-the-github_token-in-a-workflow), runs triggered by Dependabot PRs use a read-only `GITHUB_TOKEN`. Though, this can be changed by adding the needed permissions to the workflow file.

```yaml
permissions:
  contents: write
```

I was able to push an updated `dist/` directory to the Dependabot PR branch, but from there the result depended on how the branch was updated.

- Pushing a new commit would cause Dependabot to stop rebasing the branch. Depending on your use-case, this might not be a bad thing but, if you have several Dependabot PRs that you're merging, it ends up with needing to `@dependabot recreate` each one. The [docs](https://docs.github.com/en/code-security/dependabot/working-with-dependabot/managing-pull-requests-for-dependency-updates#allowing-dependabot-to-rebase-and-force-push-over-extra-commits) state you can include `[dependabot skip]` in the commit message to let Dependabot know it is allowed to force push or rebase over the commit, but in testing that seemed to instead tell Dependabot to skip this version update.
- Amending the existing commit and force pushing resulted in similar behavior, where Dependabot interpreted this as saying "skip this version update."

Either way, though, another issue comes up. When an event triggers a GitHub Actions workflow, any further events caused by that workflow run will [not trigger other workflows](https://docs.github.com/en/actions/using-workflows/triggering-a-workflow#triggering-a-workflow-from-a-workflow) (this is by design to prevent recursive invocations). This is particularly an issue if you have CI checks that must pass before a PR is merged (e.g. the codebase lints/builds correctly). When a new or amended commit is pushed to the Dependabot branch, the CI checks don't run.

This can be bypassed by providing a personal access token or GitHub App token, but I wouldn't recommend it. If your package configuration is not configured properly, running `npm package` will always result in a modified `dist/` directory, causing recursive workflow runs.

tl;dr it would be possible to set this up, but there are a few different issues involved that would result in bigger problems :(
